work/src/org/kuali/kfs/module/ar/document/ContractsGrantsInvoiceDocument.java

```diff
@@ -495,41 +495,41 @@ public String getCustomerNumber() {
         return accountsReceivableDocumentHeader.getCustomerNumber();
     }
 
-    public KualiDecimal getBudgetAmountTotal() {
+    public KualiDecimal getTotalBudgetAmount() {
         KualiDecimal total = KualiDecimal.ZERO;
 
         for (InvoiceAccountDetail accountDetail : accountDetails) {
-            total = total.add(accountDetail.getBudgetAmount());
+            total = total.add(accountDetail.getTotalBudget());
         }
 
         return total;
     }
 
-    public KualiDecimal getExpenditureAmountTotal() {
+    public KualiDecimal getTotalInvoiceAmount() {
         KualiDecimal total = KualiDecimal.ZERO;
         for (InvoiceAccountDetail accountDetail : accountDetails) {
-            total = total.add(accountDetail.getExpenditureAmount());
+            total = total.add(accountDetail.getInvoiceAmount());
         }
         return total;
     }
 
-    public KualiDecimal getCumulativeAmountTotal() {
+    public KualiDecimal getTotalCumulativeExpenditures() {
         KualiDecimal total = KualiDecimal.ZERO;
         for (InvoiceAccountDetail accountDetail : accountDetails) {
-            total = total.add(accountDetail.getCumulativeAmount());
+            total = total.add(accountDetail.getCumulativeExpenditures());
         }
         return total;
     }
 
-    public KualiDecimal getBalanceAmountTotal() {
+    public KualiDecimal getTotalBudgetRemaining() {
         KualiDecimal total = KualiDecimal.ZERO;
         for (InvoiceAccountDetail accountDetail : accountDetails) {
-            total = total.add(accountDetail.getBalanceAmount());
+            total = total.add(accountDetail.getBudgetRemaining());
         }
         return total;
     }
 
-    public KualiDecimal getBilledAmountTotal() {
+    public KualiDecimal getTotalAmountBilledToDate() {
         KualiDecimal total = KualiDecimal.ZERO;
         for (InvoiceAccountDetail accountDetail : accountDetails) {
             total = total.add(accountDetail.getAdjustedCumExpenditures());
```

Comment:
Discussed; let's rename totalAmountBilledToDate to totalPreviouslyBilled and rename adjustedCumulativeExpenditures to totalAmountBilledToDate.